_data/cloud.yml

```diff
@@ -153,7 +153,7 @@ websites:
       img: packet.png
       tfa: Yes
       sms: Yes
-      doc: https://www.packet.net/help/kb/account-management-and-billing/how-do-i-enable-two-factor-authentication-/
+      doc: https://help.packet.net/faq/the-portal
```

Comment:
According to the doc link they use software 2FA "Google Authenticator" and not SMS.